articles/quickstart/backend/_includes/_api_scopes_access_resources.md

```diff
@@ -1,8 +1,8 @@
-Scopes let you define which resources can be accessed by the user with a given Access Token. For example, you might choose to grant read access to the `messages` resource if users have the manager access level, and a write access to that resource if they have the administrator access level. 
+Permissions let you define how resources can be accessed on behalf of the user with a given access token. For example, you might choose to grant read access to the `messages` resource if users have the manager access level, and a write access to that resource if they have the administrator access level.
 
-You can add the required scopes in the **Scopes** tab of the Auth0 Dashboard's [APIs](${manage_url}/#/apis) section.
+You can define allowed permissions in the **Permissions** tab of the Auth0 Dashboard's [APIs](${manage_url}/#/apis) section.
 
-![Configure Scopes](/media/articles/server-apis/configure-scopes.png)
```

Comment:
I've just checked this and seems it's no longer being used anywhere else. Please double check, but I think we can remove it safely.

Reply:
✅